pavelib/assets.py

```diff
@@ -172,6 +196,7 @@ def compile_templated_sass(systems, settings):
     """
     for sys in systems:
         sh(django_cmd(sys, settings, 'preprocess_assets'))
+        print("\t\tFinished preprocessing assets.")
```

Comment:
Shouldn't this be outdented? We should print the message after *both* `preprocess_assets` calls have finished, not once for each system.